routing/index_graph_starter_joints.cpp

```diff
@@ -139,9 +138,12 @@ void IndexGraphStarterJoints::GetEdgeList(JointSegment const & vertex, bool isOu
   edges.clear();
 
   Segment parentSegment;
-  // В этот вектор мы сохраняем вес родительского ребра относительно всех детей.
-  // Пояснение: когда мы идем от родителя к ребенку в обратном поиске, то
-  // вычисляем вес "ребенок" -> "родитель" и для каждого ребенка этот вес будет разный.
+
+  // This vector needs for backward A* search. Assume, we have parent (p) and child_1, child_2.
```

Comment:
No need `(p)`?

`// This vector needs for backward A* search. Assume, we have a parent and child_1, child_2.`

Reply:
done